db-engine/src/test/java/com/splicemachine/db/iapi/types/SQLDecimalTest.java

```diff
@@ -49,19 +50,20 @@ public void subtractTwo() throws StandardException {
         }
         @Test
         public void serdeValueData() throws Exception {
-                UnsafeRow row = new UnsafeRow();
+                UnsafeRow row = new UnsafeRow(1);
                 UnsafeRowWriter writer = new UnsafeRowWriter(new BufferHolder(row),1);
                 SQLDecimal value = new SQLDecimal(new BigDecimal(100.0d));
```

Comment:
Did you try this with other data points? Specifically: 0,-100, Long.MAX_VALUE, Double.MAX_VALUE+1, Double.MIN_EXPONENT, etc. etc.

Seems like this would be a good case for parameterized tests to make sure we exercise the scope of decimal values.